micro-rdk/src/common/robot.rs

```diff
@@ -867,6 +881,16 @@ impl LocalRobot {
         }
         Ok(())
     }
+
+    pub fn get_cloud_metadata(&self) -> robot::v1::GetCloudMetadataResponse {
```

Comment:
Should this method return an error if these values were not available? What does RDK do?

Reply:
it should, I'll make the change